donna/donna-tests.ts

```diff
@@ -1,7 +1,7 @@
 
 
 import donna = require("donna");
-var metadata = donna.generateMetadata(['/path/to/my-module', '/path/to/another-module']);
+var metadata = donna(['/path/to/my-module', '/path/to/another-module']);
```

Comment:
:warning: Don't make test changes like this without consulting the docs for the library.

https://github.com/atom/donna#from-your-code